.github/CODEOWNERS

```diff
@@ -3,16 +3,16 @@ po/bn.po @mirwasi @sudiptachatterjee
 po/da.po @mgeisler @voss @thedataking
 po/de.po @ronaldfw @fechu
 po/el.po @Mandragorian
-po/es.po @deavid @vzz1x2 @JaviSorribes
```

Comment:
@vzz1x2, FYI! Let us know if you want to be added again so you receive emails about the Spanish translation.